.github/workflows/ci.yml

```diff
@@ -5,11 +5,27 @@ env:
 jobs:
   lint:
     runs-on: ubuntu-latest
-    container: quay.io/ovirt/vdsm-test-centos-8
+    container: quay.io/alesgar/vdsm-test-centos-8
```

Comment:
This is temporary change for testing I guess?

Reply:
Right. All changes in the ci.yml and the setup-storage script will go away. I have another branch for the test split, this branch is only for the image update. Mentioned it in the PR text.